scripts/apollo_build.sh

```diff
@@ -156,10 +156,9 @@ function main() {
         info "Running build under CPU mode on ${ARCH} platform."
     fi
     bazel_build $@
-    # Disable simulator build temporarily
-    build_simulator
     if [ -z "${SHORTHAND_TARGETS}" ]; then
         SHORTHAND_TARGETS="apollo"
+        build_simulator
```

Comment:
This might cause confusion since previously if we only use command ‘./apollo.sh build’, it means we want to build ALL modules by default.